package.json

```diff
@@ -6,10 +6,11 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "coverage-html-report": "rm -rf coverage/ && nyc report --reporter=html && opn coverage/index.html",
     "generate-constants": "./scripts/generate-constants.js",
-    "install": "node ./scripts/npm-install.js",
+    "install": "node scripts/npm-install.js",
     "rebuild-all": "npm run rebuild-core && npm run rebuild-bindings",
     "rebuild-core": "npm run submodule && node scripts/rebuild-core.js",
-    "rebuild-bindings": "node-gyp rebuild",
+    "rebuild-bindings": "node-gyp rebuild --verbose",
+    "postinstall": "node scripts/npm-postinstall.js",
```

Comment:
This is needed for windows since `deltachat.dll` from `deltachat-core-rust` is needed right next to `deltachat.node` on windows.